modules/client/src/client-common/schemas/plugin.ts

```diff
@@ -0,0 +1,78 @@
+import {
+  InvalidProposalIdError,
+  InvalidVoteValueError,
+} from "@aragon/sdk-common";
+import { array, boolean, number, object, string } from "yup";
+import { isProposalId } from "../utils";
+import {
+  addressOrEnsSchema,
+  addressSchema,
+  bigNumberishSchema,
+  daoActionSchema,
+  dateSchema,
+  ipfsUriSchema,
+} from "./common";
+import { VoteValues } from "../interfaces/plugin";
+
+export const voteValuesSchema = number().test(
+  "isVoteValue",
+  new InvalidVoteValueError().message,
+  (value) => value ? Object.values(VoteValues).includes(value) : true,
+);
+
+export const proposalIdSchema = string().notRequired().test(
+  "isProposalId",
+  new InvalidProposalIdError().message,
+  (value) => value ? isProposalId(value) : true,
+);
+
+export const proposalMetadataSchema = object({
+  title: string().required(),
+  summary: string().required(),
+  description: string().required(),
+  resources: array(
+    object({
+      url: string().required(),
+      name: string().required(),
+    }),
+  ).required(),
+  media: object({
+    header: string(),
+    logo: string(),
+  }).default(undefined),
+});
+
+export const createProposalParamsSchema = object({
+  pluginAddress: addressOrEnsSchema.required(),
+  metadataUri: ipfsUriSchema.required(),
+  actions: array(daoActionSchema).default([]),
+  startDate: dateSchema,
+  endDate: dateSchema,
+  executeOnPass: boolean(),
+  creatorVote: voteValuesSchema,
```

Comment:
@Michael-A-Heuer is this field still relevant? I think it directly collides with the fact that votes with a start date in the future cannot have any vote from the creator. 
At the same time, the creator could have no tokens and still vote